applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -129,6 +129,12 @@ config AUDIO_TEST_TONE
 	help
 	  Use button 4 to set the test tone.
 
+config AUDIO_TEST_MUTE
```

Comment:
I may see what this does later. Seems a bit odd at the moment.

Reply:
Same with AUDIO_TEST_MUTE

Reply:
See comment below